demos/browser/server.js

```diff
@@ -22,14 +23,16 @@ const indexPage = fs.readFileSync(indexPagePath);
 // Create ans AWS SDK Chime object. Region 'us-east-1' is currently required.
 // Use the MediaRegion property below in CreateMeeting to select the region
 // the meeting is hosted in.
-const chime = new AWS.Chime({ region: 'us-east-1' });
-const sts = new AWS.STS({ region: 'us-east-1' })
+const currentRegion = 'us-east-1';
```

Comment:
What's the purpose of `currentRegion` here?

Reply:
In the Demo for aws.chime it is hardcorded to use 'us-east-1' as the defualt region also in the case where we do not pass in controlRegion we will set currentRegion as controlRegion

Reply:
Can we remove `currentRegion`?